src/database/index.js

```diff
@@ -0,0 +1,41 @@
+import { Pool } from 'pg';
+import DBURL from '../../config';
+import Queries from './queries';
+
+class DBInit extends Queries {
+    constructor() {
+        super();
+        this.client = new Pool({ connectionString: DBURL });
+    }
+
+    async query(sql, data = []) {
+        try {
+            const conn = await this.client.connect();
+            console.log('Connected successfully');
+            if (data.length) {
+                const res = await conn.query(sql, data);
+                return res;
+            }
+            const result = await conn.query(sql);
+            return result;
+        } catch (err) {
+            console.log(err);
```

Comment:
Don't merge the `console.log`, remove all of them before merging

Reply:
Oh okay okay. Thanks
I fixed that